graylog-configurer

```diff
@@ -58,66 +58,69 @@ set_extractor()
   fi
 }
 
-set_admin()
-{
+set_admin() {
   username="$1"
   # Password cannot be used to login, but is needed for user creation
   password=$(cat /dev/urandom | tr -dc A-Za-z0-9 | head -c 32)
 
-  printf "\nSetup ${username} as admin\n"
```

Comment:
why don't like this one?

Reply:
What value does it add?

You already have: https://github.com/utilitywarehouse/sys-graylog-configurer/pull/16/files#diff-97ee5b8545f7d83ab7202361069f7a07R71 - for the "admin adding section"

and you have https://github.com/utilitywarehouse/sys-graylog-configurer/pull/16/files#diff-97ee5b8545f7d83ab7202361069f7a07R71 for individual accounts.

What information it provides